Makefile

```
@@ -492,7 +492,7 @@ define ci_setup_qemu_opentitan
 	# Download OpenTitan image
 	@printf "Downloading OpenTitan boot rom from: 1beb08b474790d4b6c67ae5b3423e2e8dfc9e368\n"
 	@pwd=$$(pwd) && \
-		temp=$$(mktemp -d)\
+		temp=$$(mktemp -d); \
```

```suggestion
		temp=$$(mktemp -d) && \
```

Not that I expect `mktemp` to fail, but if it does, this should short-circuit